contracts/DepositManager.sol

```diff
@@ -201,31 +205,7 @@ contract DepositManager {
         return (depositsSubTreeRoot);
     }
 
-    /**
-     * @notice Removes a deposit from the pendingDeposits queue and shifts the queue
-     * @param _index Index of the element to remove
-     * @return Remaining elements of the array
-     */
-    function removeDeposit(uint256 _index) internal {
```

Comment:
We don't need this function because we either pop the last element in the array or we nuke the whole queue. I implement the former inline and the latter in the `reset` function